admin-ui/configApiSpecs.yaml

```diff
@@ -143,13 +143,21 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/AuthenticationMethod'
+            examples:
```

Comment:
we should not check-in this yml. can we please delete it and add it entry to .gitignore